pkg/process/procutil/data_scrubber_windows.go

```diff
@@ -20,3 +25,61 @@ var (
 	// it's for handling parameters like `/p` and `/rp` in windows
 	forbiddenSymbolsRegex = "[^/a-zA-Z0-9_*]"
 )
+
+var winDotExec = []string{".com", ".exe", ".bat", ".cmd", ".vbs", ".vbe", ".js", ".jse", ".wsf", ".wsh", ".psc1", ".ps1"}
+
+// stripArguments identifies windows extension and extracts command. Otherwise, returns the first element of the cmdline before first space.
+func (ds *DataScrubber) stripArguments(cmdline []string) []string {
+	strCmdline := cmdline[0]
```

Comment:
Not sure if there's validation before this is called, but might be a good idea to add a size check before trying to access the slice with an index to avoid an out of bounds error.